src/napari_imagej/widgets/searchbar.py

```diff
@@ -54,6 +54,7 @@ def keyPressEvent(self, event):
 
     def enable(self):
         # Once the JVM is ready, allow editing
-        ensure_jvm_started()
+        init_ij()
         self.setText("")
-        self.setEnabled(True)
+        if ij():
+            self.setEnabled(True)
```

Comment:
Is the point of this function just to enable the searchbar? Let's just replace this with `lambda: self.setEnabled(True)` above?

Reply:
Well, I didn't quite do this, but I made it much simpler.